src/System.Runtime.Serialization.Primitives/ref/System.Runtime.Serialization.Primitives.cs

```diff
@@ -65,6 +64,10 @@ public sealed partial class IgnoreDataMemberAttribute : System.Attribute
     {
         public IgnoreDataMemberAttribute() { }
     }
+    public static partial class FormatterServices
```

Comment:
Why are we putting this in this contract? As soon as we add the other APIs to this type it will pull in System.Runtime.Serialization.Formatters. I thought we were going to be putting it into that contract.